external/sql/storm-sql-core/src/jvm/org/apache/storm/sql/StormSqlImpl.java

```diff
@@ -137,18 +137,12 @@ public void submit(
     }
   }
 
-  private void packageTopology(Path jar, CompilingClassLoader cl, AbstractTridentProcessor processor) throws IOException {
+  private void packageEmptyTopology(Path jar) throws IOException {
```

Comment:
Can you put some comments on why are we packaging an empty jar and where the code for executing the topology is going to be ?